pkg/cli/images.go

```diff
@@ -46,12 +47,19 @@ func (a *Image) Run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	allProjects := a.client.Options().AllProjects
 	var images []apiv1.Image
 	var image *apiv1.Image
 	tagToMatch := ""
 	repoToMatch := ""
 
+	// Image was explicitly provided
 	if len(args) == 1 {
+		c, args[0], err = parseArgGetClient(a.client, cmd, args[0])
```

Comment:
Assigning a value to an item in the `args` array seems weird to me. Might be better to declare a new variable to hold that.

Reply:
Complicates clarity too much imo, would like to keep it as is.